### DISCUSSION TITLE
About Dark mode in yun.yunyoujun.cn

### DISCUSSION THREAD

**Discussion by Eden7Ba23** [YunYouJun/hexo-theme-starry#215] [General 大局为重]
when i open link to yun.yunyoujun.cn (like link in this post https://blog.lantacn.xyz/2021/12/18/change-yun/ ) with darkmode enabled
first time it direct to the right page
but then dark mode enable and it jump to ( https://yun.yunyoujun.cn/guide/config.html#%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6 )
it really confused me :-(

**Comment by YunYouJun** [comment]
emm, i think it is a bug caused by vuepress. When dark mode, it refresehes.

**Reply by YunYouJun** [reply]
I will migrate it to vitepress. Just wait some time.
